pkg/mads/v1/service/http.go

```diff
@@ -60,8 +59,10 @@ func (s *service) handleDiscovery(req *restful.Request, res *restful.Response) {
 
 	discoveryReq.TypeUrl = mads_v1.MonitoringAssignmentType
 
-	ctx, cancel := context.WithTimeout(context.Background(), s.config.FetchTimeout)
-	defer cancel()
+	timeout := s.config.DefaultFetchTimeout
```

Comment:
Once we decide on a way to specify this client-side, all that is needed is to parse it here and pass it along.